### PR TITLE
[PLUGIN-1075] Correct functionality of skip empty data

### PR DESCRIPTION
**Pull request by itsankit-google**
[PLUGIN-1075](https://cdap.atlassian.net/browse/PLUGIN-1075)

### CODE REVIEW

File: src/main/java/io/cdap/plugin/google/sheets/source/utils/MultipleRowRecord.java [hunk 1]
```diff
@@ -99,7 +99,7 @@ private boolean convertToSingleRow(Map<String, ComplexMultiValueColumn> headered
         CellData cellData;
         if (headerValue.getData().size() > index) {
           cellData = headerValue.getData().get(index);
-          isEmptyRow = cellData == null;
+          isEmptyRow = isEmptyRow && (cellData == null || cellData.isEmpty());
```
Comment (sechegaray) [inline src/main/java/io/cdap/plugin/google/sheets/source/utils/MultipleRowRecord.java:109]: Could we add a comment to this method explaining that it returns a boolean if the row is empty? (And that it converts the row into a result row).

File: widgets/GoogleSheets-batchsource.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
           "label": "Modification Date Range",
           "name": "modificationDateRange",
           "widget-attributes": {
-            "default": "today",
+            "default": "lifetime",
```
Comment (sechegaray) [inline widgets/GoogleSheets-batchsource.json]: This change seems unrelated?

**Review by sechegaray — CHANGES_REQUESTED** [review]

File: src/main/java/io/cdap/plugin/google/sheets/source/utils/MultipleRowRecord.java (hunk 1 above)
Comment (itsankit-google) [reply src/main/java/io/cdap/plugin/google/sheets/source/utils/MultipleRowRecord.java:109]: ack

File: widgets/GoogleSheets-batchsource.json (hunk 2 above)
Comment (itsankit-google) [reply widgets/GoogleSheets-batchsource.json]: Agree not related to skip empty data config but was a small change so added in the same PR. Should I rollback this change and raise another PR for it?

**Review by sechegaray — APPROVED** [review]
